### PR TITLE
[TIMOB-23140] Android: Add lineSpacing to Label

### PR DESCRIPTION
**Pull request by m1ga**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-23140

adding lineSpacing to multiline Labels
http://developer.android.com/reference/android/widget/TextView.html#setLineSpacing%28float,%20float%29

### CODE REVIEW

File: android/titanium/src/java/org/appcelerator/titanium/TiC.java [hunk 1]
```diff
@@ -707,6 +707,21 @@
 	/**
 	 * @module.api
 	 */
+	public static final String PROPERTY_LINE_SPACING = "lineSpacing";
```
Comment (ashcoding) [inline android/titanium/src/java/org/appcelerator/titanium/TiC.java]: Could you add the Strings in alphabetical order?

**Comment by ashcoding** [comment]
Everything looks good, just would like the TiC properties to be arranged alphabetically. :) Thank you @m1ga

**Comment by m1ga** [comment]
sure, its below property_lines now

**Comment by hansemannn** [comment]
Approved! Thank you @m1ga, great work! :rocket:

**Pull request merged**